Assets/AltTester/Runtime/Input.cs

```diff
@@ -155,6 +156,37 @@ private void FixedUpdate()
         }
 
     }
+    public static GameObject ExecuteHierarchy<T>(GameObject root, BaseEventData eventData, EventFunction<T> callbackFunction) where T : IEventSystemHandler
+    {
+        IList<Transform> s_InternalTransformList = new List<Transform>();
+        GetEventChain(root, s_InternalTransformList);
+
+        var internalTransformListCount = s_InternalTransformList.Count;
+        for (var i = 0; i < internalTransformListCount; i++)
+        {
+            var transform = s_InternalTransformList[i];
+            if (Execute(transform.gameObject, eventData, callbackFunction))
+                if ((transform ?? false) && (transform.gameObject ?? false))
+                    return transform.gameObject;
+                else
+                    return null;
+
+        }
+        return null;
+    }
+    private static void GetEventChain(GameObject root, IList<Transform> eventChain)
+    {
+        eventChain.Clear();
+        if (root == null)
```

Comment:
Shouldn't you use `gameObject ?? false` here as well?

Reply:
no, because there is no posibility to be in a situation where the root is destroyed